rest/processing/process.py

```diff
@@ -165,15 +184,29 @@ def id_to_data_collection(self, collection_id):
     def get_node_by_process_id(self, process_id):
         return get_node_by_process_id(self.process_graph, process_id)
 
-    def get_collection(self):
-        load_collection_node = self.get_node_by_process_id("load_collection")
-        return self.id_to_data_collection(load_collection_node["arguments"]["id"])
+    def get_all_load_collection_nodes(self):
+        return get_all_load_collection_nodes(self.process_graph)
+
+    def get_collections(self):
+        collections = {}
+        load_collection_nodes = self.get_all_load_collection_nodes()
+        for node_id, load_collection_node in load_collection_nodes.items():
+            from_time, to_time = self.get_temporal_extent(load_collection_node)
+            data_collection = self.id_to_data_collection(load_collection_node["arguments"]["id"])
+            collections[f"node_{node_id}"] = {
+                "data_collection": data_collection,
+                "from_date": from_time,
+                "to_date": to_time,
+            }
+
+        return collections
 
     def get_bounds_from_load_collection(self):
         """
         Returns bbox, EPSG code, geometry
         """
-        load_collection_node = self.get_node_by_process_id("load_collection")
+        load_collection_nodes = list(self.get_all_load_collection_nodes().values())
+        load_collection_node = load_collection_nodes[0]
```

Comment:
Since data fusion only accepts 1 bbox / set of bounds for all collections, I think this should be mentioned somewhere.

I also think there should be some warning / error if the bounds of `load_collection` processes are different, since only the bounds of the first `load_collection` process in the list will be used, other will be ignored.

Reply:
Added a comment explaining this and also added a check which would now raise an error (might need better wording) stating that spatial extents for all collections should be equal

Reply:
Thanks, the check and the comment are perfect.

The wording is not the best, since collections can have different spatial extent, it's just that the extents should be the same in all `load_collection` processes.
Maybe something like

Data fusion is possible only if all load_collection processes have the same spatial extent

Reply:
Yeah this is what I was looking for, changed.